MainPanel.c

```diff
@@ -19,10 +19,8 @@ void MainPanel_updateTreeFunctions(MainPanel* this, bool mode) {
    FunctionBar* bar = MainPanel_getFunctionBar(this);
    if (mode) {
       FunctionBar_setLabel(bar, KEY_F(5), "Sorted");
-      FunctionBar_setLabel(bar, KEY_F(6), "Collap");
    } else {
       FunctionBar_setLabel(bar, KEY_F(5), "Tree  ");
-      FunctionBar_setLabel(bar, KEY_F(6), "SortBy");
```

Comment:
Why remove those?

Reply:
As I these lines changes the default bar. But there's no reason to do that anymore because the label is always set to `SortBy`